security/acme-client/pkg-descr

```diff
@@ -15,6 +15,7 @@ Added:
 * add status for accounts to backend and WebGUI
 * add button to manually trigger account registration
 * add support for All-Inkl.com domain API (#1130)
+* add support for deSEC.io domain API
```

Comment:
Well, it was not added in version 2.0 :) But I can fix this post-merge.

Reply:
I created a 2.1 section :)